components/morph.js

```diff
@@ -1,29 +1,29 @@
-import { create_float_buffer, create_index_buffer } from './context';
-import { Entity } from './entity';
+import { Component } from '../core';
+import { create_float_buffer, create_index_buffer } from '../core/context';
```

Comment:
Nit: `../core` also re-exports everything from `../core/context` so you can put these imports in the same line as the `Component` import.